packages/api/src/utils/logger.ts

```diff
@@ -133,10 +131,8 @@ const truncateObjectDeep = (object: any, length: number): any => {
 
 class GcpLoggingTransport extends LoggingWinston {
   log(info: any, callback: (err: Error | null, apiResponse?: any) => void) {
-    const sizeInfo = jsonStringify(info).length
-    if (sizeInfo > MAX_LOG_SIZE) {
-      info = truncateObjectDeep(info, 500) as never // the max length for string values is 500
-    }
```

Comment:
@jacksonh this is the change where we reduce the size of logs by truncating any string value to 500 char; e.g. `content` in gql request and sql parameters